Makefile

```diff
@@ -6,7 +6,6 @@ _GID ?= 10001
 
 VENV := $(shell echo $${VIRTUAL_ENV-.venv})
 INSTALL_STAMP = $(VENV)/.install.stamp
-POETRY_VIRTUALENVS_IN_PROJECT = true
```

Comment:
Ahh, so this wasn't doing what I thought it was doing 😅

Reply:
😊 indeed